### DISCUSSION TITLE
Symlinks on windows with Alldebrid and Rclone not working

### DISCUSSION THREAD

**Discussion by DavidOBE** [rogerfar/rdt-client#460] [General]
**What version are you using?**
v2.0.62

**Wat OS are you running?**
Windows 10

**Are you using Docker or as a service?**
Service

**Which debrid provider are you using?**
Alldebrid

**Which downloader are you using?**
Symlinks

**Please attach a log file here with the log setting set to debug**
See attached
[rdtclient_010.log](https://github.com/rogerfar/rdt-client/files/14379856/rdtclient_010.log)

Torrents are seen in RDT Client.
Files can be played in the mounted rclone drive.
However, the torrent stays at "Queued for downloading" and the folder in the download path are not symlinks but empty folders.

Not sure what I am missing to make it work.

Thanks!

**Comment by DavidOBE** [comment]
I was wondering, is it because rdt do not look inside alldebrid folder?

**Comment by DavidOBE** [comment]
Update: I have mapped the webdav using netdrive and now rdtclient pick it up and create the symlinks. However, Plex doesn't see the file, so not related to the tool

Not sure why rclone didn't work. I used the command that alldebrid website asked to use:
rclone.exe mount alldebrid:magnets X: --dir-cache-time 10s --multi-thread-streams=0 --cutoff-mode=cautious --vfs-cache-mode minimal --network-mode --buffer-size=0 --read-only

Even if I manage to make rclone work, that will probably not make plex pick up the symlinks anyway...

**Comment by cesarcassio** [comment]
The problem is that Rdtclient or ARR apps create symlinks for files instead of creating them for folders. This works in Jellyfin but not in Plex. Plex can find every file in C:\mnt\remote(it's symlinks) \ _ _all_ _ because _ _all_ _ it's not a symlink

**Reply by vpierre1776** [reply]
Same issue here, with plex
